src/System.IO.Pipelines/System/IO/Pipelines/FlushResult.cs

```diff
@@ -7,6 +7,21 @@ public struct FlushResult
     {
         internal ResultFlags ResultFlags;
```

Comment:
Do we still set this directly? Make it private if not.

Reply:
We do.